app/helpers/application_helper.rb

```diff
@@ -10,7 +10,7 @@ module ApplicationHelper
   include StiRoutingHelper
   include ToolbarHelper
   include TextualSummaryHelper
-  include NumberHelper
+  ActionView::Helpers::NumberHelper.prepend NumberHelper # override rails number helper with our needs for number_to_human_size
```

Comment:
Previously, we included the rails number helper first in the controller...

then, we included ours in the application helper which was then included in the controller.

This change ensures we prepend our NumberHelper on the rails ones and do it directly in the application helper to ensure we're not including the original one directly.